.dockerignore

```
@@ -0,0 +1,12 @@
+
+__pycache__/
```

I'd probably go for:
```suggestion
**/__pycache__
```
This is mostly an excuse to mention this because I don't expect people to necessarily be aware, but `.dockerignore` has slightly different behaviour to `.gitignore`, in particular in that it _doesn't_ ignore subfolders by default - we need to explicitly reference them with the `**` prefix to mean arbitrary levels of nesting should be considered for this rule. I think that your other existing entries wouldn't matters here, but this is one of those issues that can easily lead to weird confusion when some behaviour starts caching oddly!